airgun/entities/host_new.py

```diff
@@ -975,6 +975,22 @@ def refresh_applicability(self, entity_name):
         self.browser.plugin.ensure_page_safe()
         view.dropdown.item_select('Refresh applicability')
 
+    def read_update_value(self, entity_name, key, value):
```

Comment:
Could you please change the wording of the function name.
```suggestion
    def read_updated_value(self, entity_name, key, value):
```

Reply:
Updated the function name.